identity-issuance/src/main/java/com/android/identity/issuance/proofing/defaultGraph.kt

```diff
@@ -56,6 +56,9 @@ fun defaultGraph(
                     whenNotAuthenticated {}
                 }
             }
+            on(id = "eId", text = "Yes, derive the document from PIN-protected German eID") {
```

Comment:
I would use something like "GermanEidCard" as the ID here.

Reply:
Done

Reply:
done